mailbox/api/src/main/java/org/apache/james/mailbox/MailboxListener.java

```diff
@@ -170,8 +170,27 @@ public MailboxPath getMailboxPath() {
          */
         private static final long serialVersionUID = 1L;
 
-        public MailboxDeletion(MailboxSession session, MailboxPath path) {
+        private final QuotaRoot quotaRoot;
+        private final QuotaCount quotaCount;
+        private final QuotaSize quotaSize;
+
+        public MailboxDeletion(MailboxSession session, MailboxPath path, QuotaRoot quotaRoot, QuotaCount quotaCount, QuotaSize quotaSize) {
```

Comment:
I'd be more confortable with a name like `QuotaUpdateUponMailboxDeletion` (because here I 'm being confused with MailboxDeleted)...